bin/varnishd/cache/cache_filter.h

```diff
@@ -33,6 +33,12 @@ struct req;
 struct vfp_entry;
 struct vfp_ctx;
 
+enum vrt_filter_err {
+	FLT_OK = 0,
+	FLT_LOOKUP = -1,	// already found / not found
+	FLT_NAME_TAKEN = -2,
+};
```

Comment:
I'm not fond of the names that I don't find very descriptive, I would rather avoid using one entry for two different cases.

For example:

- ok
- duplicate (already added)
- conflict (other filter with same name present)
- missing (cannot remove)